scripts/fill_dcp1_file_metadata/fill_dcp1_metadata.py

```diff
@@ -0,0 +1,188 @@
+import requests as rq
+import argparse
+import json
+import logging
+
+from requests.exceptions import HTTPError
+
+from hca_ingest.api.ingestapi import IngestApi
+
+"""
+The script follows the next steps to fix the missing size and fileContentType for DCP1 datasets:
+
+1. Check the project UUID provided is valid 
+2. Retrieve all data file metadata from azul
+3. Parse and massage the data to avoid DCP/1 / HCA Release / DCP/2 analysis files (Not in ingest)
+4. For each of the metadata files:
+   a. Check metadata is not already filled (Priority: Ingest > Azul)
+   b. PUT metadata file
+"""
+INGEST_TOKEN = ""
```

Comment:
read token from environment variable or a CL argument
```suggestion
import os
INGEST_TOKEN = os.environ['INGEST_TOKEN']
```

Reply:
I generally prefer [os.getenv](https://docs.python.org/3/library/os.html#os.getenv):
```
INGEST_TOKEN  = os.getenv('INGEST_TOKEN')
```

As this will not **fail** if the environment variable is missing.